cli/compose/types/types.go

```diff
@@ -78,9 +78,10 @@ type Config struct {
 type ServiceConfig struct {
 	Name string
 
-	CapAdd          []string `mapstructure:"cap_add"`
-	CapDrop         []string `mapstructure:"cap_drop"`
-	CgroupParent    string   `mapstructure:"cgroup_parent"`
+	Build           BuildConfig `mapstructure:"build"`
```

Comment:
not a problem, but technically you don't need the mapstructure annotation here, since it defaults to build